README.md

```diff
@@ -79,25 +79,22 @@ Above you can find the example app links for the _latest_ HERE SDK version. If y
 
 ## What You Need to Execute the Example Apps
 1. Acquire a set of credentials by registering yourself on [developer.here.com](https://developer.here.com/) - or ask your HERE representative.
-2. Download the latest HERE SDK artifacts for your desired platform. These can be found on [developer.here.com](https://developer.here.com/) unless otherwise noted.
+2. Download the latest HERE SDK artefact for your desired platform. These can be found on [developer.here.com](https://developer.here.com/) unless otherwise noted.
 3. Please refer to the minimum requirements and supported devices as listed in our _Developer Guide_.
```

Comment:
1. Acquire a set of credentials: Follow the steps from the _Developer Guide_ for your HERE SDK edition.
2. Download the latest HERE SDK package for your desired platform as shown in the  _Developer Guide_.
3. ...